sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -298,29 +246,9 @@ sender.sendMessage(message);
 
 #### Receive messages from a session
 
-Receivers can fetch messages from a specific session or the first available, unlocked session.
+Receiving messages from sessions is similar to receiving messages from a non session enabled queue or subscription. The difference is in the builder and the class you use.
 
-<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L175-L181 -->
-```java
-// Creates a session-enabled receiver that gets messages from the session "greetings".
-ServiceBusSessionReceiverAsyncClient sessionReceiver = new ServiceBusClientBuilder()
-    .connectionString("<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>")
-    .sessionReceiver()
-    .queueName("<< QUEUE NAME >>")
-    .buildAsyncClient();
-Mono<ServiceBusReceiverAsyncClient> receiverAsyncClient = sessionReceiver.acceptSession("greetings");
-```
-
-<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L188-L194 -->
-```java
-// Creates a session-enabled receiver that gets messages from the first available session.
-ServiceBusSessionReceiverAsyncClient sessionReceiver = new ServiceBusClientBuilder()
-    .connectionString("<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>")
-    .sessionReceiver()
-    .queueName("<< QUEUE NAME >>")
-    .buildAsyncClient();
-Mono<ServiceBusReceiverAsyncClient> receiverAsyncClient = sessionReceiver.acceptNextSession();
-```
+In non session case, you would use the `ServiceBusProcessorClient` by using the sub builder `processor()`. In case of sessions, you would use the `ServiceBusSessionProcessorClient` by using the sub builder `sessionProcessor()`. You can pass the maximum number of sessions you want the processor to process concurrently as well.
```

Comment:
Both session and non-session builders will create the same processor client type instance.

```suggestion
In non-session case, you would use the sub builder `processor()`. In case of sessions, you would use the sub builder `sessionProcessor()`. Both sub builders will create an instance of `ServiceBusProcessorClient` configured to work on a session or a non-session Service Bus entity. In the case of the session processor, you can pass the maximum number of sessions you want the processor to process concurrently as well.
```